### PR TITLE
chore: only show runs which are linkable

### LINKED ISSUE #3478: Alerts Activity: Only show runs which are linkable
We currently show all the task/alert runs, out of which, only the alert runs are linkable and not the custom task ones. So, in order to make this more usable and relevant, only show the alerts runs so that we only show the linkable runs.

### PR DESCRIPTION
**Pull request by subirjolly**
Closes #3478 

### BEFORE
image

### AFTER
image

### CODE REVIEW

File: src/types/checks.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export type CheckIDsMap = {
+  [_: string]: boolean
+}
```
Comment (hoorayimhelping) [inline src/types/checks.ts:3]: small nit, but for consistency I think: 
```suggestion
  [key: string]: boolean
```
is preferred. If you regex search the codebase for `\[_:` no matches come up. If you search for `\[key:` you'll see that this is a much more common pattern

File: src/types/checks.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export type CheckIDsMap = {
+  [_: string]: boolean
```
Comment (hoorayimhelping) [inline src/types/checks.ts:2]: small nit, but for consistency I think: 
```suggestion
  [key: string]: boolean
```
is preferred. If you regex search the codebase for `\[_:` no matches come up. If you search for `\[key:` you'll see that this is a much more common pattern

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 53 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3497)

File: src/types/checks.ts (hunk 2 above)
Comment (subirjolly) [reply src/types/checks.ts]: Makes total sense. I saw `x` instead of `key`.. so, i chose `_`

File: src/alerting/utils/activity.ts [hunk 3]
```diff
@@ -7,20 +7,24 @@ import {event} from 'src/cloud/utils/reporting'
...
 
 export const runAlertsActivityQuery = (
   orgID: string,
+  checkIDs: CheckIDsMap,
   {offset, limit, since}: LoadRowsOptions
 ): CancelBox<StatusRow[]> => {
   const start = since ? Math.round(since / 1000) : '-1d'
 
+  // Empty checkIDs set will result in 0 records being fetched by the following query
+  const checks = Object.keys(checkIDs).join('", "')
```
Comment (hoorayimhelping) [inline src/alerting/utils/activity.ts:22]: 👍 for pulling this out into it's own line. It being inline with the query made my dumb brain hurt.

**Review by hoorayimhelping — APPROVED** [review]
lgtm!

File: src/alerting/utils/activity.ts (hunk 3 above)
Comment (subirjolly) [reply src/alerting/utils/activity.ts:22]: thank you for your review, sir!

**Pull request merged**